spec/features/course/unread_status_management_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Announcements read/unread status management', type: :feature do
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:first_user) { create(:user, role: :administrator) }
+    let(:second_user) { create(:user, role: :administrator) }
+    let(:course) { create(:course) }
+
+    describe 'sidebar' do
+      let!(:announcement) { create(:course_announcement, course: course, creator: second_user) }
+
+      before do
+        login_as(first_user)
+        visit course_achievements_path(course)
+      end
+
+      it 'shows the right unread number' do
+        expect(Course::Announcement.get_unread_number(first_user, course)).to eq(1)
+        expect(page).to have_link('Announcements (1)')
+      end
+    end
+
+    describe 'when visit announcement' do
+      describe 'index page' do
+        let!(:announcements) { create_list(:course_announcement, 5, course: course,
+                                           creator: second_user) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 30, 66 should be on its own line.